src/test/java/io/strimzi/test/container/AbstractIT.java

```diff
@@ -41,7 +41,9 @@ protected static Stream<Arguments> retrieveKafkaVersionsFile() throws IOExceptio
     }
 
     protected void assumeDocker() {
-        Assumptions.assumeTrue(System.getenv("DOCKER_CMD") == null || "docker".equals(System.getenv("DOCKER_CMD")));
+        Assumptions.assumeTrue(
+            System.getenv("DOCKER_CMD") == null ||
+                ("docker".equals(System.getenv("DOCKER_CMD")) || "podman".equals(System.getenv("DOCKER_CMD"))));
```

Comment:
doesn't it mean - always true?

Reply:
Maybe having just `System.getenv("DOCKER_CMD") == null` should be okay? I don't want to be more strict (i.e., 
```java
Assumptions.assumeTrue(
    System.getenv("DOCKER_CMD") != null &&
        ("docker".equals(System.getenv("DOCKER_CMD")) || "podman".equals(System.getenv("DOCKER_CMD"))));
```

Reply:
Hard to say, what is the reason for this? It is for `docker` only tests? Or `docker` has to be installed ?

Reply:
> It is for docker only tests? Or docker has to be installed ?

Both I think.

Reply:
Let's move the discussion bellow.